Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,42 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+        // Homework Lesson 1 & 2 - SilentKingdom
+            // BMI Calculator for 2 people
+
+            // First person = variable1
+            Console.Write("What is your name? ");
+            string name1 = Console.ReadLine();
+            Console.Write("And how old are you? ");
+            int age1 = int.Parse(Console.ReadLine());
+            Console.Write("How much do you weigh (kg)? ");
+            float weight1 = float.Parse(Console.ReadLine());
+            Console.Write("How tall are you? (cm) ");
+            float height1 = float.Parse(Console.ReadLine());
+            // Testing string + variable layouts
+            Console.WriteLine("\n{0} is {1}, and weighs {2}kg at a height of {3}cm.\n", name1, age1, weight1, height1);
+
+            // Their BMI
+            float heightConvert1 = height1 / 100;
+            float height_s1 = heightConvert1 * heightConvert1;
```

Comment:
It's not super important, but you might want to take a look at variable naming conventions in C# https://github.com/ktaranov/naming-convention/blob/master/C%23%20Coding%20Standards%20and%20Naming%20Conventions.md

Reply:
Thanks for that, I'll try to stick with naming conventions in the future.